### PR TITLE
Adding save button...

### PR DESCRIPTION
**Pull request by bbusagroningen**
save message, hover properties, enable user to only add new connections if both source and destination are selected

### CODE REVIEW

File: Switchboard/public/app/components/connections/ConnectionController.js [hunk 1]
```diff
@@ -17,14 +17,24 @@ define(['shared/Registry', 'components/connections/ConnectionFactory', 'shared/p
      * connection data to the title of the page
      */
     var ConnectionController = function ($uibModal, $scope, $routeParams, ConnectionFactory, PageFactory) {
-
+        
+        var changesSaved = "All modifications have been saved &#10004;";
+        var changesMade= "Modifications have not yet been saved &#10006;";
```
Comment (Carolien989) [inline Switchboard/public/app/components/connections/ConnectionController.js]: nice feature!

File: Switchboard/public/app/components/connections/NewLinkedFieldModalController.js [hunk 2]
```diff
@@ -16,8 +16,13 @@ define(['shared/Registry'], function (Registry) {
             destination: undefined
         }
         $scope.submit = function () {
+            if($scope.select.source === undefined || $scope.select.destination === undefined){
+                alert("One of the fields are empty!");
+                return;
```
Comment (Carolien989) [inline Switchboard/public/app/components/connections/NewLinkedFieldModalController.js]: works on my laptop!

**Comment by killje** [comment]
Have you tried it on a smaller screen if it still looks correctly

**Comment by killje** [comment]
Looks good now

**Pull request merged**